crates/holochain/tests/sharded_gossip/mod.rs

```diff
@@ -476,171 +476,164 @@ async fn mock_network_sharded_gossip() {
                         gossip,
                     } => {
                         if let kitsune_p2p::GossipModuleType::ShardedRecent = module {
-                            if let GossipProtocol::Sharded(gossip) = gossip {
-                                use kitsune_p2p::gossip::sharded_gossip::*;
-                                match gossip {
-                                    ShardedGossipWire::Initiate(Initiate { intervals, .. }) => {
-                                        // Capture the intervals from alice.
-                                        // This works because alice will only initiate with one simulated
-                                        // agent at a time.
-                                        last_intervals = Some(intervals);
-                                        let arc = data.agent_to_arc[&agent];
-                                        let agent_info = data.agent_to_info[&agent].clone();
-                                        let interval = arc;
-
-                                        // If we have info for alice check the overlap.
-                                        if let Some(alice) = &alice {
-                                            let a = alice.storage_arc;
-                                            let b = interval.clone();
-                                            debug!("{}\n{}", a.to_ascii(10), b.to_ascii(10));
-                                            let a: DhtArcSet = a.inner().into();
-                                            let b: DhtArcSet = b.inner().into();
-                                            if !a.overlap(&b) {
-                                                num_missed_gossips += 1;
-                                            }
+                            let GossipProtocol::Sharded(gossip) = gossip;
```

Comment:
Would it be better to just leave this as-is, in case we ever do make a new protocol, we'd end up deleting this indent, then adding it back in again later for no reason? I don't feel strongly, so if you want to keep this change, that's fine too.

Reply:
Yeah, I only did this because clippy said so, but I think I agree with you more.